src/components/Input/Input.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { InputHTMLAttributes, FC } from 'react';
+import { classNames } from '../../utils/classNames';
+import Icon from '../Icon/Icon';
+
+interface IInputProps {
+  /** set customized style */
+  className?: string;
+  /** set input bar size */
+  size?: 'mini' | 'small' | 'large' | 'big' | 'huge' | 'massive';
```

Comment:
try to not use size direactly, you could use inputSize

Reply:
done